### DISCUSSION TITLE
How do I deploy Authentik in a way that allows for an equivalent of "multi-master replication" in OpenLDAP land?

### DISCUSSION THREAD

**Discussion by dkowis** [goauthentik/authentik#6516] [Q&A] [answered]
Trying to use the right words, so that if someone else searches for this kind of functionality, they'll find this.

I want to deploy more than one "instance" of authentik. I can run it on my faster infrastructure, and then on a Pi4, for if the faster infrastructure is offline, for whatever reason.

How do I ensure that they both stay in-sync and function well? Does that matter? is it as simple as a DNS load balancer and postgresql/redis replication?

Thanks!

**Comment by BeryJu** [comment, marked as answer]
authentik itself doesn't have any special requirements for this, you can have as many server and worker instances as you want to. The only requirements for this are HA Postgres and HA Redis (allthough there are currently some issues with Redis since sentinel is not supported (yet)), and then loadbalance with whatever method you want

**Reply by dkowis** [reply]
I suspected as much, but I wasn't sure! Thanks for clarifying!
